src/style.css

```diff
@@ -1 +1,59 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-size: 12px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
```

Comment:
set fonts to html selctor and add fallbak fonts
<img width="252" alt="image" src="https://user-images.githubusercontent.com/66749919/200027789-6410c511-41a9-4398-a0a3-494b7ba4d7b8.png">